app/assets/stylesheets/application.scss

```diff
@@ -183,15 +195,70 @@ ul {
 }
 
 .recipe-body {
+  background-color: rgba(94, 94, 94, 0.7);
+  border-style: solid;
```

Comment:
Properties should be ordered background-color, border-color, -moz-border-radius, -webkit-border-radius, border-radius, border-style, color, list-style-type, margin-left, margin-right, padding, padding-bottom, text-align, width